### DISCUSSION TITLE
Any love for android?

### DISCUSSION THREAD

**Discussion by hapishyguy** [amidaware/tacticalrmm#1953] [General]
Can this control android or can we make it to work with android.

Thanks

**Comment by silversword411** [comment]
It works 'ish https://play.google.com/store/search?q=meshcentral&c=apps

But based on the last couple years, android and ios are both making it harder and harder to do. 57 clicks, switch to permission manager, grant permission, close/relaunch to get it. bigger and bigger PITA territory. Just adding more barriers to make it non-practical to provide support.

**Reply by ederpardeiro** [reply]
Is there a way to display in RMM Dashboard?

**Reply by hapishyguy** [reply]
Yes I totally understand those permissions issue on android. Its getting harder every time. Its because of the scammers who are taking the source e code and injecting with malware to steal data. I would not refrain saying that mostly these people are from asian side. It needs to be said.

What I actually need is that I have android boxes. These are streaming boxes. These boxes are not on current android OS. These boxes have eg android 10 or 11 and are rooted. The OS do not update automatically. I can tell the vendor to pre install any apps. So for me I need to access the box without any clicking on the permissions by the end user in case they have any issue. Right now if there is any iasue and they need help I have to tell them to click on so many permissions in order to get access to the box and almost 70% of the time they click on wrong prompts by default and deny those or miss those or Get annoyed. If there is a way where I can have some remote access for android where there are no permissions to be clicked by the end user it would be great. I can have it embed in the rom by the vendor.
